src/settings-ui/Settings.UI/Controls/SettingsGroup/SettingsGroup.xaml

```diff
@@ -19,50 +18,51 @@
             <Setter.Value>
                 <ControlTemplate TargetType="controls:SettingsGroup">
                     <Grid HorizontalAlignment="Stretch">
-                        <VisualStateManager.VisualStateGroups>
-                            <VisualStateGroup x:Name="CommonStates">
-                                <VisualState x:Name="Normal"/>
-                                <VisualState x:Name="Disabled">
-                                    <VisualState.Setters>
-                                        <Setter Target="HeaderPresenter.Foreground" Value="{ThemeResource TextFillColorDisabledBrush}" />
-                                        <Setter Target="DescriptionPresenter.Foreground" Value="{ThemeResource TextFillColorDisabledBrush}" />
-                                    </VisualState.Setters>
-                                </VisualState>
-                            </VisualStateGroup>
-                        </VisualStateManager.VisualStateGroups>
                         <Grid.RowDefinitions>
-                            <RowDefinition Height="Auto"/>
-                            <RowDefinition Height="Auto"/>
-                            <RowDefinition Height="Auto"/>
+                            <RowDefinition Height="Auto" />
+                            <RowDefinition Height="Auto" />
+                            <RowDefinition Height="Auto" />
                         </Grid.RowDefinitions>
-                        <TextBlock x:Name="HeaderPresenter"
-                                   Text="{TemplateBinding Header}"
-                                   Grid.Row="0"
-                                   Style="{ThemeResource BodyStrongTextBlockStyle}"
-                                   Margin="1,32,0,0"
-                                   AutomationProperties.HeadingLevel="Level2"/>
+                        <TextBlock
+                            x:Name="HeaderPresenter"
+                            Grid.Row="0"
+                            Margin="1,32,0,0"
```

Comment:
Why the "1"?
(also for following)

Reply:
The lightgray border of the SettingsCard is 1 pixel.. visually the header text should align with the background of the cards, else it looks misaligned.